cypress/e2e/po/edit/ingress.po.ts

```diff
@@ -0,0 +1,27 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import NameNsDescription from '@/cypress/e2e/po/components/name-ns-description.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+
+export default class IngressEditPagePo extends PagePo {
```

Comment:
individual resource pages basically contain a cypress/e2e/po/edit/resource-detail.po.ts. that PO wraps some of the inner components likely to appear on the page.

for this use case would suggest a pattern like cypress/e2e/po/edit/machines.po.ts (though the name saveCreateForm is odd, sounds like one of my mistakes)

Reply:
I've imported the resource-detail.po to get access to the save via cruResource.